styles/global.scss

```diff
@@ -41,6 +41,47 @@ img {
   margin: 30px;
 }
 
+.imagebutt {
+  background-color: #242424;
+  border:none;
+}
+
+.imagebutt:hover {
+  cursor:pointer;
+}
+
+.mainbuttons {
+  display:flex;
+  justify-content: flex-start;
+  margin: 20px;
+  width: 30vw;
+  height: 10vh;
+}
+
+.textbox {
+  display:flex;
+  justify-content: flex-start;
+  margin: 20px;
+  width: 30vw;
+  height: 10vh;
+}
+
+.healthBar {
+  display: flex;
+  align-items: center;
+}
+
+.healthSection {
```

Comment:
again a picky comment - css the norm is to use the kebab-case instead of camelCase